pkg/kustomize/kustomize_cli.go

```diff
@@ -49,3 +54,31 @@ func extractSemanticVersion(version string) (string, error) {
 	}
 	return regex.FindString(version), nil
 }
+
+// ContainsKustomizeConfig finds out if there is any kustomize resource in the cwd or subdirectories
+func (k *KustomizeCLI) ContainsKustomizeConfig(dir string) bool {
+	if len(k.FindKustomizationYamlPaths(dir)) != 0 {
+		return true
+	}
+
+	return false
+}
+
+// FindKustomizationYamlPaths looks for the kustomization.yaml i.e. kustomize resources in present and sub-directories
+func (k *KustomizeCLI) FindKustomizationYamlPaths(dir string) (resource []string) {
+	fp, err := filepath.Abs(dir)
+	var resources []string
+	err = filepath.Walk(fp, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if strings.Contains(path, "kustomization.yaml") {
+			resources = append(resources, path)
+		}
+		return nil
+	})
+	if err != nil {
```

Comment:
Not sure whether logging is the best approach here. Usually, one returns the error, but let's see